packages/cli/internal/pkg/cli/context/manager_deploy_test.go

```diff
@@ -55,7 +56,10 @@ func TestManager_Deploy(t *testing.T) {
 			},
 		},
 		"context error": {
-			expectedErr: fmt.Errorf("context 'testContextName1' is not defined in Project 'testProjectName' specification"),
+			expectedErr: actionable.NewError(
+				fmt.Errorf("context 'testContextName1' is not defined in Project 'testProjectName' specification"),
+				"Please add the context to your project spec and deploy it or specify a different context from the command 'agc context list'",
```

Comment:
Looks like this message is duplicated in multiple places. I think this duplication should be eliminated.

Reply:
It's only used once in the core package and four times during testing. I'll create a constant for the testing package.

Reply:
Updated the associated tests